src/components/App.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
It's best-practice to put the router in it's own file named Routes.js

https://betterprogramming.pub/8-basic-and-advanced-react-router-tips-6993ece8f57a

Reply:
OK, guess I didn't get what you meant by your comment before. Will fix it.